### PR TITLE
Simple implementation of copy input to clipboard (Ctrl+C)

### PR DESCRIPTION
**Pull request by daandeheij**
The ability to copy input strings to the clipboard was requested in #10955. This is a very simple implementation using Ctrl+C, as suggested by @janisozaur.

I wanted to include a simple acknowledgement that the text is copied (as this is not very clear as selecting text is not possible). I have used the `context_show_error` in this PR, but I am not sure whether this is the most appropriate way. Of course, this message can also be omitted.   

The following screenshot shows the `context_show_error`. Note that the string in this commit is `copied text to clipboard`, contrary to this screenshot (I changed the string after taking the screenshot).

image

### CODE REVIEW

**Comment by tupaschoal** [comment]
Can you add a screenshot of what the message looks like to the PR description, please?

In any case, I'm wondering whether we need an information pop-up or not.

**Comment by daandeheij** [comment]
I just added a screenshot. I think ideally would be to have some sort of pop-up/message box notify the user that the text was copied, and then to disappear on its own. This current message sticks around until the next key/mouse press, which is slightly inconvenient.

File: src/openrct2-ui/TextComposition.cpp [hunk 1]
```diff
@@ -175,6 +175,13 @@ void TextComposition::HandleMessage(const SDL_Event* e)
                     CursorRight();
                     console.RefreshCaret();
                     break;
+                case SDLK_c:
+                    if (modifier & KEYBOARD_PRIMARY_MODIFIER && _session.Length)
```
Comment (janisozaur) [inline src/openrct2-ui/TextComposition.cpp]: Just to be explicit about evaluation order here, I would suggest adding parentheses, as done in other such constructs around:
```suggestion
                    if ((modifier & KEYBOARD_PRIMARY_MODIFIER) && _session.Length)
```

**Comment by AaronVanGeffen** [comment]
Could you add a changelog entry as well? (We just released v0.2.5 tonight, so it'll have to go under v0.2.5+.)

**Comment by daandeheij** [comment]
Added a changelog entry just now under v.0.2.5+

**Comment by duncanspumpkin** [comment]
@daandeheij in future have as your commit message `Fix #<number of issue>: description...` That way it will auto close the issue on merge.

**Pull request merged**